### PR TITLE
Upgrade to Fastify v3

### PR DESCRIPTION
**Pull request by thomheymann**
audit fix

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [x] commit message and code follows [Code of conduct](https://github.com/fastify/fastify/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

File: test/multipart.test.js [hunk 1]
```diff
@@ -17,6 +18,8 @@ const crypto = require('crypto')
 
 const filePath = path.join(__dirname, '../README.md')
 
+// const test = () => { };
```
Comment (mcollina) [inline test/multipart.test.js:21]: please drop this.

File: test/multipart.test.js [hunk 2]
```diff
@@ -1,5 +1,6 @@
 'use strict'
 const os = require('os')
+// const t = require('tap');
```
Comment (mcollina) [inline test/multipart.test.js:3]:
```suggestion
```

File: package.json [hunk 3]
```diff
@@ -3,42 +3,41 @@
...
     "eslint-config-standard": "^14.0.0",
-    "eslint-config-standard-with-typescript": "^8.0.0",
+    "eslint-config-standard-with-typescript": "^16.0.0",
     "eslint-plugin-import": "^2.17.2",
-    "eslint-plugin-node": "^10.0.0",
+    "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-promise": "^4.1.1",
     "eslint-plugin-typescript": "^0.14.0",
-    "fastify": "^2.0.0",
+    "fastify": "^3.0.0-alpha.1",
```
Comment (Ethan-Arrowood) [inline package.json:26]: Use `rc`

File: test/append-body.test.js [hunk 4]
```diff
@@ -427,114 +427,114 @@ test('addToBody with shared schema', t => {
...
-      encoding: '7bit',
-      filename: 'README.md',
-      limit: false,
-      mimetype: 'text/markdown'
-    }])
-
-    reply.send('ok')
-  })
+    }, function (req, reply) {
+      console.log('POST')
```
Comment (mcollina) [inline test/append-body.test.js:443]: Can you remove this?

File: test/append-body.test.js [hunk 5]
```diff
@@ -427,114 +427,114 @@ test('addToBody with shared schema', t => {
...
-
-  fastify.post('/', {
-    schema: {
-      body: {
-        type: 'object',
-        required: ['myField', 'myFile'],
-        properties: {
-          myField: { type: 'string' },
-          myFile: { type: 'array', items: 'mySharedSchema#' }
+  }).then(() => {
```
Comment (mcollina) [inline test/append-body.test.js:430]: You can use async/await instead.

Comment (thomheymann) [reply test/append-body.test.js:430]: done

**Comment by mcollina** [comment]
@Ethan-Arrowood can you take a look?

**Pull request merged**